### PR TITLE
added missing pt-BR translation

### PR DESCRIPTION
**Pull request by GuilhermeZaniniMoreira**

### CODE REVIEW

File: kofta/public/locales/pt-BR/translation.json [hunk 1]
```diff
@@ -84,10 +84,10 @@
       "followers": "seguidores",
       "following": "seguindo",
       "followHim": "seguir",
+      "unfollow": "deixar de seguir",
       "followingHim": "seguindo",
-      "copyProfileUrl": "copy profile url",
-      "urlCopied": "URL copied to clipboard",
-      "unfollow": "Unfollow"
```
Comment (HarrisonMayotte) [inline kofta/public/locales/pt-BR/translation.json:90]: You removed this one but didn't readd it with the translation.

**Review by HarrisonMayotte — CHANGES_REQUESTED** [review]
See below comment.

File: kofta/public/locales/pt-BR/translation.json (hunk 1 above)
Comment (GuilhermeZaniniMoreira) [reply kofta/public/locales/pt-BR/translation.json:90]: unfollow key? check line 87. I changed the key order to match the same as english

Comment (HarrisonMayotte) [reply kofta/public/locales/pt-BR/translation.json:90]: Oh I missed that. All good then. I'll merge this.

**Review by HarrisonMayotte — APPROVED** [review]

**Pull request merged**